crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -255,6 +437,63 @@ def proposals
   api :GET, "/crowbar/:barclamp/1.0/proposals/template",
     "Returns the content of a proposal template"
   param :barclamp, String, desc: "Name of the barclamp", required: true
+  example '
+  {
+    "id": "template-dns",
```

Comment:
You cannot provide information through the API that is not being documented.
In other words, you cannot expect attributes or fields are being consumed by a client if they're not documented somewhere.